tests/testthat/test-Design-class.R

```diff
@@ -317,3 +317,76 @@ test_that("DualResponsesSamplesDesign user constructor arguments names are as ex
     ordered = TRUE
   )
 })
+
+# DADesign ----
+
+test_that(".DADesign works as expected", {
+  # Create an instance of .DADesign
```

Comment:
looks nice, good work! Can you just remove the comments? (see above where we also don't have them)
(Generally, we avoid comments where not absolutely necessary, as we intend to have readable code instead.)